generic/tests/nfs_corrupt.py

```diff
@@ -150,6 +172,13 @@ def is_service_active(self):
         else:
             return False
 
+    def iptables_rule_gen(self):
```

Comment:
Generator is a bit overkill for and kind of hard to read.
Better to take iptables_rule_gen(self, op='A'), and use if ...clause...to increase readability .